views/publication/pages/add.templ

```diff
@@ -10,108 +10,128 @@ import (
 templ Add(c *ctx.Ctx, step int) {
```

Comment:
![Screenshot 2024-08-08 at 09 46 55](https://github.com/user-attachments/assets/2b56b8c7-133a-4fb1-b253-defd67d54cd3)

There seems to be a gap on the right from the page,
and an extra vertical line. Compare to https://biblio-backend-ugent-prototype.vercel.app/biblio/researcher/add-publication/add-publication.html

Reply:
Fixed in bc5b1e418092ea7d6cbb414966c623b504a72454